ChopChop/Components/Date+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension Date {
+    static let today: Date = Calendar.current.startOfDay(for: Date())
+
+    var startOfDay: Date {
+        Calendar.current.startOfDay(for: self)
+    }
+}
+
+// convert seconds to hh:mm:ss
+func get_HHMMSS_Display(seconds: Int) -> String {
```

Comment:
Small matter but this should probably be named something like `getHHMMSSDisplay` or `getFormattedDisplay`